src/test/java/uk/gov/pay/api/filter/RateLimiterFilterTest.java

```diff
@@ -59,24 +60,23 @@ public void setup() {
     @Test
     public void shouldCheckRateLimitsWhenFilterIsInvoked() throws Exception {
         when(mockContainerRequestContext.getMethod()).thenReturn("POST");
-        
+
         rateLimiterFilter.filter(mockContainerRequestContext);
-        
+
         verify(rateLimiter).checkRateOf(eq(ACCOUNT_ID), any());
     }
 
     @Test
     public void shouldSendErrorResponse_whenRateLimitExceeded() throws Exception {
         doThrow(RateLimitException.class).when(rateLimiter).checkRateOf(eq("account-id"), any());
 
-        try {
-            rateLimiterFilter.filter(mockContainerRequestContext);
-        } catch (WebApplicationException e) {
-            Response response = e.getResponse();
-            assertEquals(429, response.getStatus());
-            assertEquals("application/json", response.getHeaderString("Content-Type"));
-            assertEquals("utf-8", response.getHeaderString("Content-Encoding"));
-            assertEquals("{\"code\":\"P0900\",\"description\":\"Too many requests\"}", response.getEntity());
-        }
+        WebApplicationException webApplicationException = assertThrows(WebApplicationException.class,
```

Comment:
So we can assert an exception is thrown and then carry on doing more stuff without even needing a try/catch block? Neat!